src/main/java/core/basesyntax/impl/PrepareData.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.Preparer;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class PrepareData implements Preparer {
```

Comment:
Bad name + see image provided in task description, this class shoudl create report (repost is a string with data)